### DISCUSSION TITLE
Allow sorting/filtering requests by user

### DISCUSSION THREAD

**Discussion by phlurblepoot** [sct/overseerr#1545] [Feature Requests]
Would be awesome if we could sort the requests page by user for those that have the boys requests permission. Would make readability easier in certain situations.

**Comment by cleme29** [comment]
yes that would be helpful when I make batches for certain users !

**Reply by TheCatLady** [reply]
This is already possible.  Go to the user's profile, and click "Recent Requests."  Alternatively, navigate to the user list and click the user's request count.

**Reply by dgriffinzero** [reply]
Interesting.  I've been using Overseerr for months and didn't think it could be done because it's not a filter option on the Requests screen.  If it can't/won't be added to the Requests page, can it be added to the FAQ?

**Reply by shark2k** [reply]
> This is already possible. Go to the user's profile, and click "Recent Requests." Alternatively, navigate to the user list and click the user's request count.

I think it would be beneficial to have the ability to do the filter on the actual request page.
Say you have a lot of requests that you are looking through on the requests page and you notice there are a couple of users that have more requests then others.  Honestly, it's kind of foolish to have to go the users profile or another page to get to their recent requests at that point.

The better option would be on the requests page to have a filter that is dynamically populated based on the other filters.  This way if you have 30 current unfulfilled requests from a total of 5 users and that is what you are looking at, you can then filter on the user but it would only be one of those 5 users for the current requests you are looking at.

-Shark2k

**Reply by TheCatLady** [reply]
> I think it would be beneficial to have the ability to do the filter on the actual request page.

Planned; adding it to the user page was just low-hanging fruit and thus implemented first.
